var/spack/repos/builtin/packages/spatial-index/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class SpatialIndex(PythonPackage):
+    """Spatial indexer for geometries and morphologies"""
+
+    homepage = "http://www.example.com"
+    git      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
```

Comment:
```suggestion
    git      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
    url      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
```

Makes it easier to use `diy` with fantasy versions.